src/straight_skeleton_2.cpp

```diff
@@ -0,0 +1,56 @@
+
+#include "straight_skeleton_2.h"
+#include <CGAL/Polygon_2.h>
+#include <CGAL/create_straight_skeleton_2.h>
+
+typedef CGAL::Exact_predicates_inexact_constructions_kernel K;
+typedef K::Point_2 Point;
+typedef CGAL::Polygon_2<K> Polygon_2;
+typedef CGAL::Straight_skeleton_2<K> Ss;
+typedef boost::shared_ptr<Ss> SsPtr;
+typedef CGAL::Straight_skeleton_2<K>::Halfedge_const_handle Halfedge_const_handle;
+typedef CGAL::Straight_skeleton_2<K>::Vertex_const_handle Vertex_const_handle;
+
+Edges pmp_create_interior_straight_skeleton(
+    Eigen::Ref<const compas::RowMatrixXd> &V)
+{
+    Polygon_2 poly;
+    for (int i = 0; i < V.rows(); i++)
+    {
+        poly.push_back(Point(V(i, 0), V(i, 1)));
+    }
+    assert(poly.is_counterclockwise_oriented());
```

Comment:
perhaps it is worth checking what happens with these assertions on the Python side

Reply:
i removed the assert